stdlib/Dates/src/io.jl

```diff
@@ -530,14 +530,40 @@ end
 # show
 
 function Base.show(io::IO, dt::DateTime)
+    y,m,d = yearmonthday(dt)
+    h = hour(dt)
+    min = minute(dt)
+    s = second(dt)
+    ms = millisecond(dt)
+    if ms == 0
+        print(io, "DateTime(",y,", ",m,", ",d,", ",h,", ",min,", ",s,")")
+    else
+        print(io, "DateTime(",y,", ",m,", ",d,", ",h,", ",min,", ",s,", ",ms,")")
```

Comment:
I think doing `"DateTime($y, $m, $d, $h, $min, $s, $ms)"` would be easier for reading the code. Also `mi` is the typical variable name for minutes.

Reply:
You are right. I will make the changes right away.